src/test/java/io/vertx/core/FutureTest.java

```diff
@@ -1424,4 +1424,133 @@ public void testFailureNotification() {
     promise.fail(failure);
     await();
   }
+
+  @Test
+  public void testToCompletionStageTrampolining() {
```

Comment:
Not mandatory but we could save some duplication by refactoring these tests

Reply:
I think they are readable as-is

Reply:
ok